store/localcachelayer/emoji_layer_test.go

```diff
@@ -80,7 +80,7 @@ func TestEmojiStoreCache(t *testing.T) {
 		mockStore.Emoji().(*mocks.EmojiStore).AssertNumberOfCalls(t, "Get", 2)
 	})
 
-	t.Run("first call by id force no cached, second not cached, third cached", func(t *testing.T) {
+	t.Run("first call by name force no cached, second not cached, third cached", func(t *testing.T) {
```

Comment:
> 

@migbot  I see this inconsistency in almost all the test files under store/localcachelayer.  I see `not cached` and sometimes `no cached`. 

I found the  `id` instead of `name` fixed here to be definitely misleading in-case this  test failed, and wanted to make sure its fixed. 

I would be in favor of doing another PR with the clean up  in all the test files and consistently use `not cached` instead of the `no cached`. I would be open to include those changes here if you strongly feel that it would be fine for them to be included in this PR.

Reply:
so in my head the options are: 

1.  Do a separate PR for all that cleanup in all the test files including this one, and leave this fix to be just about id/name change
2.  Cleanup this file, and then do a PR for the other test files 
3. Just do all the cleanup in all the test files this PR

I will be in favor of doing option 1 Or 3. Can be convinced to do 2

Reply:
I think i prefer 1. Lets merge this, and create another PR fixing other cases where is need.